tpcx_bb/xbb_tools/utils.py

```diff
@@ -235,6 +234,15 @@ def write_clustering_result(result_dict, output_directory="./", filetype="csv"):
 
     return 0
 
+def remove_benchmark_files():
```

Comment:
Can we make a note of this behavior?

Perhaps this and the record read time benchmark runner config are good reasons to start a `configuration.md` file explaining logging options?

Reply:
Agreed, Created https://github.com/rapidsai/tpcx-bb/issues/20 to track this.

Reply:
I will do it in a follow up pr